Content.Client/Atmos/UI/GasAnalyzerWindow.xaml.cs

```diff
@@ -0,0 +1,115 @@
+using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Localization;
+using Robust.Shared.Maths;
+using Robust.Shared.Utility;
+using Content.Shared.Temperature;
+using static Content.Shared.Atmos.Components.SharedGasAnalyzerComponent;
+
+namespace Content.Client.Atmos.UI
+{
+    /// <summary>
+    /// Gas analyzer UI.
+    /// </summary>
+    [GenerateTypedNameReferences]
+    public partial class GasAnalyzerWindow : SS14Window
+    {
+        int _lastGasCount = 0;
+
+        public GasAnalyzerWindow()
+        {
+            RobustXamlLoader.Load(this);
+        }
+        public void UpdateState(GasAnalyzerBoundUserInterfaceState state)
+        {
+            GasGrid.RemoveAllChildren();
+
+            var statusMessage = new FormattedMessage();
+            StatusLabel.SetMessage(statusMessage);
+
+            if (state.Error != null)
+            {
+                GasBar.RemoveAllChildren();
+                statusMessage.PushColor(Color.Red);
+                statusMessage.AddMarkup(Loc.GetString("gas-analyzer-window-error-text", ("errorText", state.Error)));
+                statusMessage.Pop();
+                return;
+            }
+
+            statusMessage.AddMarkup(Loc.GetString("gas-analyzer-window-pressure-text", ("pressure", $"{state.Pressure:0.##}")));
+            statusMessage.PushNewline();
+            statusMessage.AddMarkup(Loc.GetString("gas-analyzer-window-temperature-text",
```

Comment:
In the future, I think it'd be a good idea to annotate these with warning/alert symbols when they're out of safe limits. Maybe it could use the same data source as https://github.com/space-wizards/space-station-14/pull/5018?